google_compute_engine/ip_forwarding/tests/ip_forwarding_utils_test.py

```diff
@@ -20,25 +20,33 @@
 from google_compute_engine.test_compat import unittest
 
 
+def _create_mock_successful_process():
```

Comment:
If we are going to introduce a function for creating a process, I think the function should be more generally usable  (e.g. can take parameters for return code and return value). If you would like to make that update, there are also a couple other style nits (for consistency). Otherwise, you can also inline the three lines in the two places where the function is currently called.